bcipy/helpers/tests/test_task.py

```diff
@@ -51,12 +53,67 @@ def test_alphabet_images(self):
              '_'])
 
 
-class TestTrialReshaper(unittest.TestCase):
+class TestDataReshaper(unittest.TestCase):
+    """Checks that `data_reshaper` is a transparent wrapper of `trial_reshaper` and `inquiry_reshaper`"""
+
+    def setUp(self):
+        self.target_info = ['first_pres_target', 'fixation',
+                            'target', 'nontarget', 'nontarget']
+        self.timing_info = [1.001, 1.2001, 1.4001, 1.6001, 1.8001]
+        self.n_channel = 7
+        self.fs = 256
+        self.trial_per_inquiry = 3
+        self.eeg = np.random.randn(self.n_channel, self.fs * 10)
```

Comment:
It looks like you create eeg data for this file a few times. It may be worth pulling it up into a test method and calling in the unittests

Reply:
I put this into a follow-up ticket: https://www.pivotaltracker.com/story/show/178294216

Reply:
Ok - I put this into a follow-up ticket: https://www.pivotaltracker.com/story/show/178302572